src/app/modules/submission/email-submission/email-submission.controller.ts

```diff
@@ -257,7 +262,8 @@ const submitEmailModeForm: RequestHandler<
         // This is why sendSubmissionToAdmin is separated from sendEmailConfirmations in 2 blocks
         return MailService.sendSubmissionToAdmin({
           replyToEmails: EmailSubmissionService.extractEmailAnswers(
-            parsedResponses,
+            // TODO: A better solution might exist
+            [...parsedResponses.responses, ...parsedResponses.ndiResponses],
```

Comment:
can always expose a function in the ParsedResponsesService for this with all the various checks you need